question-service/repository/category-repository.js

```diff
@@ -1,4 +1,5 @@
 const db = require('../models')
+const { Op } = require('sequelize')
```

Comment:
Curious what's the meaning of "Op"?

Reply:
Sequelize operators, some examples include Op.and, Op.or, Op.in and etc.